unfoldingWord/tc-create-app
# highlight error fixed for greek elision

## Describe what your pull request addresses

- This  Fixes [#179](https://github.com/unfoldingWord/scripture-resources-rcl/issues/179), by making the normalizer non-greedy as it is for uw-quote-helpers normalizer.

## Test Instructions
- Open tCC, select en_tn and select book mat 12:30
- you can see all the words are highlighted
